colink/protocol.py

```diff
@@ -75,13 +77,37 @@ def run(self, cl: CoLink = None):
             threads.append(t)
         for t in threads:
             t.start()
-        while True:
-            if q.empty():
-                if threading.active_count() == 1:
-                    break
-            else:
-                err = q.get()
-                raise err
+        if keep_alive_when_disconnect:
+            while True:
+                if q.empty():
+                    if threading.active_count() == 1:
+                        break
+                else:
+                    err = q.get()
+                    raise err
```

Comment:
so we don't need to sleep(0.001) here (similar to the one below)? Also, can we have it longer, like 0.1s for example?

Reply:
Sure, I will add a sleep here.